doc/install/manual_install.rst

```diff
@@ -0,0 +1,128 @@
+  :doc:`FreeSurfer <freesurfer>`.
+
+Python IDEs
+===========
+
+Most users find it convenient to write and run their code in an `Integrated
+Development Environment`_ (IDE). Some popular choices for scientific
+Python development are:
+
+- `Spyder`_ is a free and open-source IDE developed by and for scientists who
+  use Python. It is included by default in the ``base`` environment when you
+  install Anaconda, and can be started from a terminal with the command
+  ``spyder`` (or on Windows or macOS, launched from the Anaconda Navigator GUI).
+  It can also be installed with `dedicated installers <https://www.spyder-ide.org/#section-download>`_.
+  To avoid dependency conflicts with Spyder, you should install ``mne`` in a
+  separate environment, like explained in the earlier sections. Then, set
+  Spyder to use the ``mne`` environment as its default interpreter by opening
+  Spyder and navigating to
+  :samp:`Tools > Preferences > Python Interpreter > Use the following interpreter`.
+  There, paste the output of the following terminal commands::
+
+      $ conda activate mne
+      $ python -c "import sys; print(sys.executable)"
+
+  It should be something like ``C:\Users\user\anaconda3\envs\mne\python.exe``
+  (Windows) or ``/Users/user/opt/anaconda3/envs/mne/bin/python`` (macOS).
+
+  If the Spyder console can not start because ``spyder-kernels`` is missing,
+  install the required version in the ``mne`` environment with the following
+  commands in the terminal::
+
+      $ conda activate mne
+      $ conda install spyder-kernels=HERE_EXACT_VERSION -c conda-forge
+
+  Refer to the `spyder documentation <https://docs.spyder-ide.org/current/troubleshooting/common-illnesses.html#spyder-kernels-not-installed-incompatible>`_
+  for more information about ``spyder-kernels`` and the version matching.
+
+  If the Spyder graphic backend is not set to ``inline`` but to e.g. ``Qt5``,
+  ``pyqt`` must be installed in the ``mne`` environment.
+
+- `Visual Studio Code`_ (often shortened to "VS Code" or "vscode") is a
+  development-focused text editor that supports many programming languages in
+  addition to Python, includes an integrated terminal console, and has a rich
+  ecosystem of packages to extend its capabilities. Installing
+  `Microsoft's Python Extension
+  <https://marketplace.visualstudio.com/items?itemName=ms-python.python>`__ is
+  enough to get most Python users up and running. VS Code is free and
+  open-source.
+- `Atom`_ is a text editor similar to vscode, with a package ecosystem that
+  includes a `Python IDE package <https://atom.io/packages/ide-python>`__ as
+  well as `several <https://atom.io/packages/atom-terminal>`__
+  `packages <https://atom.io/packages/atom-terminal-panel>`__
+  `for <https://atom.io/packages/terminal-plus>`__
+  `integrated <https://atom.io/packages/platformio-ide-terminal>`__
+  `terminals <https://atom.io/packages/term3>`__. Atom is free and open-source.
```

Comment:
I know you just moved things here but I think that Atom is the past and I would remove it from here